packer_test/core_tests/suite_test.go

```diff
@@ -0,0 +1,23 @@
+package core_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/packer/packer_test/lib"
+	"github.com/stretchr/testify/suite"
+)
+
+type PackerCoreTestSuite struct {
+	*lib.PackerTestSuite
+}
+
+func Test_PackerPluginSuite(t *testing.T) {
+	baseSuite, cleanup := lib.PackerCoreSuite(t)
```

Comment:
I don't understand the need for PackerCoreTestSuite over lib.PackerTestSuite and how they relate or differ from lib.PackerCoreSuite(). Is this level of abstraction offering a benefit that may not be immediately noticeable?

Reply:
So the difference between the two is mostly that the tests run are not the same, being two separate packages/instances of TestSuite means that both can run in parallel. They also may not have the same needs for plugins for example, so one will have to compile a bunch of plugins, and others don't.
As for the names right now, I picked something somewhat randomly, I'm completely open to rename them to something else if you think it's confusing, no problems here!